<?xml version="1.0"?>
<launch>

	<rosparam command="load" file="$(find order_package)/config/waypoints.yaml" />
	<rosparam command="load" file="$(find order_package)/config/store_locations.yaml" />


    <arg name="product_data_path" default="$(find order_package)/data/products.csv" />
    <node pkg="order_package" name="product_param_loader" type="product_param_loader.py" clear_params="true">
            <param name="product_data_path" value="$(arg product_data_path)" />
    </node>

    <arg name="order_handler_namespace" default="order_node" />
    <node pkg="order_package" name="$(arg order_handler_namespace)" type="order_node.py" output="screen" />

</launch>
